cartography/intel/aws/ecr.py

```diff
@@ -139,6 +142,21 @@ def cleanup(neo4j_session: neo4j.Session, common_job_parameters: Dict) -> None:
     run_cleanup_job('aws_import_ecr_cleanup.json', neo4j_session, common_job_parameters)
 
 
+def _get_image_data(boto3_session: boto3.session.Session, region: str, repositories: List[Dict]) -> Dict:
```

Comment:
```suggestion
def _get_image_data(boto3_session: boto3.session.Session, region: str, repositories: List[Dict[str, Any]]) -> Dict[str, Any]:
```

while you're in here might as well add the extra chars